docs/source/releases/v1.13.0a0.rst

```diff
@@ -9,7 +9,6 @@
  | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
  | # # # for more details. If you did not receive the license, for more information see:
  | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
-
 Version 1.13.0a0 (2024-04-24)
```

Comment:
Did we mean to rename this file from ``v1_13_0a0.rst`` to ``v1.13.0a0.rst``?

Reply:
This was my mistake. Fixed.